lib/fireblocks/api/transactions.rb

```diff
@@ -24,58 +27,81 @@ def create(options)
         end
 
         def from_vault_to_external(
-          amount:,
+          amount: nil,
           asset_id:,
           source_id:,
-          destination_id:,
-          one_time_address:,
+          destination: nil,
+          destinations: nil,
+          destination_id: nil,
+          one_time_address: nil,
           tag: nil,
           options: {}
         )
-          one_time_address_hash = {
-            address: one_time_address
-          }
-          one_time_address_hash.merge(tag: tag) if tag
+
+          destination_params = destination_payload(
+            destination_type: EXTERNAL_WALLET,
```

Comment:
We should have amount in here, too?

Reply:
> We should have amount in here, too?

When a single `destination` is passed, the [`amount` within the body](https://github.com/fundamerica/fireblocks/pull/8/files#diff-a8b3d610914c61a56f835f880f5877adf5c37af2dda0bd445ecc9276c1729fb3L41) should be valid and the same as previous logic/post params.

When multiple `destinations`, I updated the API-side to send the sum for total amount, but also the amount per destination object within the array ([TransactionRequestDestination](https://docs.fireblocks.com/api/#transactionrequestdestination)) will be included per AssetTransfer.